### PR TITLE
update v2 docs

### PR DESCRIPTION
**Pull request by tomfrenken**
## What Has Changed?

Since the execute and executeRaw APIs change in v2, this PR contains the tabs approach to show both examples.

### CODE REVIEW

File: docs/js/features/connectivity/destination.mdx [hunk 1]
```diff
@@ -216,10 +218,26 @@ The lookup process done by the SAP Cloud SDK involves the following steps:
 If no token is given or the destination is not found in the subscriber account the provider account is used as a fallback.
 To control this fallback behavior a selection strategy can be passed to the destination lookup:
 
+<Tabs groupId="version" defaultValue="one" values={[
+{ label: 'Version 1.X', value: 'one'},
+{ label: 'Version 2.X', value: 'two'}]}>
+
+<TabItem value="one">
```
Comment (artemkovalyov) [inline docs/js/features/connectivity/destination.mdx:225]: Let's adopt the same Tabs naming and implementation as [here](https://github.com/SAP/cloud-sdk/blob/main/docs/js/release-policy.mdx). Then we'll have it uniform across our docs and changing versions will be a simple "find and replace". The value will universally stay the same which simplifies maintenance further. That's where unification will help a lot to my conviction.

**Review by artemkovalyov — CHANGES_REQUESTED** [review]
Please, take labels and values as they are in release policy. It will let us easier maintain things at scale in the future.

File: docs/js/features/connectivity/destination.mdx (hunk 1 above)
Comment (tomfrenken) [reply docs/js/features/connectivity/destination.mdx]: I agree, I've updated the docs with your approach 👍

File: docs/js/features/connectivity/destination.mdx [hunk 2]
```diff
@@ -216,10 +218,27 @@ The lookup process done by the SAP Cloud SDK involves the following steps:
 If no token is given or the destination is not found in the subscriber account the provider account is used as a fallback.
 To control this fallback behavior a selection strategy can be passed to the destination lookup:
 
+
+<Tabs groupId="version" defaultValue="major" values={[
```
Comment (artemkovalyov) [inline docs/js/features/connectivity/destination.mdx:222]:
```suggestion
<Tabs groupId="version" defaultValue="deprecated" values={[
```

File: docs/js/features/connectivity/destination.mdx [hunk 3]
```diff
@@ -247,13 +266,30 @@ In every request, these destinations are added to the destinations returned by t
 There are situations where you do not have a JWT issued by the XSUAA but need to look a destination up e.g. in background processes.
 In such situations the property `iss` of the `DestinationAccessorOptions` can be used:
 
+<Tabs groupId="version" defaultValue="major" values={[
```
Comment (artemkovalyov) [inline docs/js/features/connectivity/destination.mdx:269]:
```suggestion
<Tabs groupId="version" defaultValue="deprecated" values={[
```

**Review by artemkovalyov — APPROVED** [review]
LGTM, let's keep default value as deprecated before the release of v2

**Pull request merged**